app/views/waste_exemptions_engine/edit_cancelled_forms/new.html.erb

```diff
@@ -0,0 +1,26 @@
+
+<div class="column-two-thirds">
+<% if @edit_cancelled_form.errors.any? %>
+
+  <h1 class="heading-large"><%= t(".error_heading") %></h1>
+
+  <% @edit_cancelled_form.errors.full_messages.each do |message| %>
+    <li><%= message %></li>
+  <% end %>
+
+<% else %>
+
+  <%= form_for(@edit_cancelled_form) do |f| %>
+    <%= render("waste_exemptions_engine/shared/errors", object: @edit_cancelled_form) %>
```

Comment:
```suggestion
```

Reply:
This had the same issues as the EditCompleteForm from the other PR. So I've done a matching fix: https://github.com/DEFRA/waste-exemptions-engine/pull/142/commits/4c00561259db806438981d2cec23a3b222b90268